packages/xo-server/src/xo-mixins/cloud-configs.mjs

```diff
@@ -45,6 +45,10 @@ export default class {
     return this._db.get()
```

Comment:
should you filter out the networ cloud config ?

Reply:
The name of the method is `getAllCloudConfigs` and not `getAllUserConfigs`.
So if a user call this method in CLI, he expect to get all cloud configs.
I added the filter in the UI.